stable/yugabyte/values.yaml

```
@@ -92,6 +92,7 @@ serviceEndpoints:
     ## Sets the Service's externalTrafficPolicy
     # externalTrafficPolicy: ""
     app: "yb-master"
+    loadBalancerIP: ""
```

```suggestion
    # loadBalancerIP: ""
```
How about keeping this commented out?